psydac/linalg/block.py

```diff
@@ -106,6 +109,13 @@ def pads( self ):
     def n_blocks( self ):
         return len( self._spaces )
 
+    @property
+    def connectivity( self ):
+        return self._connectivity
+
+    def get_connectivity( self, i, j ):
+        return self._connectivity[i,j]
+
```

Comment:
This becomes simply
```suggestion
    @property
    def connectivity( self ):
        return self._connectivity_readonly

```

Reply:
fixed in [7426fa5](https://github.com/pyccel/psydac/pull/213/commits/7426fa50b3353eb00a528005c2fb2be977f6a433)